kiwi/system/setup.py

```diff
@@ -864,11 +865,20 @@ def _import_custom_scripts(self):
                 ]
             )
 
+    @staticmethod
+    def _is_file_executable(file_name):
```

Comment:
I didn't know that `os.access` exists, so why not use that one directly? `Path.access` would only call that function anyway.

Reply:
That's a good point, just verified `os.access` does not throw an exception if the file does not exist, just returns `False`. Probably would make sense to make a dedicated method just to handle this sort of little cases.

When I suggested the method I imagined a method that checks existence first and then the access mode, this way we could easily have a method that reacts different than just providing `False` in case the file does not exist, we could throw an exception or not according to some optional parameter. 

Said that I would also be fine by just calling `os.access` method directly and just do not use any other method.

Reply:
> That's a good point, just verified os.access does not throw an exception if the file does not exist, just returns False.

While that's technically correct, it's unexpected behavior for a python function. I'll create a sane wrapper for that.

Reply:
Done.